docs/includes/topic_cert-manager.rst

```diff
@@ -55,7 +55,7 @@ Configuration
 
 .. code-block:: text
 
-    $ sudo emacs /etc/neutron/services/f5/f5-openstack-agent.ini
+    $ sudo vi /etc/neutron/services/f5/f5-openstack-agent.ini
```

Comment:
Me too! I did it very reluctantly. Unfortunately I've found that emacs won't always work in my OpenStack environments, but vi always will. (sobs)